example/android/app/build.gradle

```diff
@@ -175,7 +169,7 @@ android {
             //signingConfig signingConfigs.debug
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            signingConfig signingConfigs.release
+            signingConfig signingConfigs.debug
```

Comment:
Shouldn't this be in release mode?

Reply:
this is to make debug apk
releas apk mostly use to upload for playstore , developer can make  change as per mobile app requirement